oauthproxy.go

```diff
@@ -115,6 +117,53 @@ func NewFileServer(path string, filesystemPath string) (proxy http.Handler) {
 	return http.StripPrefix(path, http.FileServer(http.Dir(filesystemPath)))
 }
 
+func CreateSecureProxy(opts *Options, validator func(string) bool) http.Handler {
```

Comment:
In golang, it's conventional to name constructor-like functions with `New`, like `NewSecureProxy`.

Reply:
at least in this file, the convention is `func New.*` versus `func.Create.*`. Couldn't find anything specific to Golang though. @timothy-spencer